activerecord/lib/active_record/encryption/compressor.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module ActiveRecord
+  module Encryption
+    # The algorithm used for compressing and uncompressing data.
+    #
+    # It uses Zlib by default. If you want to use a different compressor, you can
+    # implement the +compress+ and +uncompress+ methods in a module and pass it to
```

Comment:
```suggestion
    # implement the +compress+ and +decompress+ methods in a module and pass it to
```

Reply:
Sorry I was confused by the naming changes, let's make sure they're all consistently `(un)compress` like @flavorjones suggested